cohere_aws/client.py

```diff
@@ -210,6 +211,197 @@ def create_endpoint(
             model.deploy(n_instances, instance_type, endpoint_name=endpoint_name)
         self.connect_to_endpoint(endpoint_name)
 
+    def chat(
+        self,
+        message: str,
+        # should only be passed for stacked finetune deployment
+        model: Optional[str] = None,
+        # should only be passed for Bedrock mode; ignored otherwise
+        model_id: Optional[str] = None,
+        preamble: Optional[str] = None,
+        chat_history: Optional[List[Dict[str, Any]]] = None,
+        temperature: Optional[float] = None,
+        max_tokens: Optional[int] = None,
+        stream: Optional[bool] = False,
+        p: Optional[float] = None,
+        k: Optional[float] = None,
+        tools: Optional[List[Dict[str, Any]]] = None,
+        tool_results: Optional[List[Dict[str, Any]]] = None,
+        search_queries_only: Optional[bool] = None,
+        documents: Optional[List[Dict[str, Any]]] = None,
+        prompt_truncation: Optional[str] = None,
+        raw_prompting: Optional[bool] = False,
+        return_prompt: Optional[bool] = False,
+        variant: Optional[str] = None,
+    ) -> Union[Chat, StreamingChat]:
+        """Returns a Chat object with the query reply.
+
+        Args:
+            message (str): The message to send to the chatbot.
+
+            stream (bool): Return streaming tokens.
```

Comment:
can the actual param order match the docstring order? it's confusing to follow like this